spark/spark-defaults.conf.in

```diff
@@ -0,0 +1,35 @@
+#
```

Comment:
@fvillanustre as part of the spark-hpcc integration effort, we'll be supplying spark config files dynamically configured. However those files include the following license. Are we ok to do this?